src/style.css

```diff
@@ -1,3 +1,119 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background: #fff;
+}
+
+.card__image {
+  display: block;
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin: 0 0 4px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: #060b35;
+}
+
+.card__code {
+  margin-bottom: 16px;
+}
+
+.group-1 {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+}
+
+.review-count {
+  color: #060b35;
+}
+
+.price-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.price-label {
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.price {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: bold;
+  color: #060b35;
+}
+
+.card__buy {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  box-sizing: border-box;
+  height: 40px;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+
+  font-size: 14px;
+  line-height: 16px;
+  color: #fff;
+  font-weight: 700;
+  text-decoration: none;
+  text-transform: uppercase;
+
+  background-color: #00acdc;
+}
+
+.card__buy:hover {
+  color: #00acdc;
+  background-color: #fff;
+}
+
+.stars {
```

Comment:
it'd be nice if you create such files as `card.scss`, `stars.scss`, `variables.scss`, `main.scss`
move styles for card to `card.scss`
move styles for stars to `stars.scss`
move variables for stars to `variables.scss`
import those files to `main.scss`